api/src/main/java/au/org/aodn/nrmn/restapi/service/SpreadSheetService.java

```diff
@@ -42,150 +38,47 @@
 
     @Autowired
     private S3IO s3client;
+    
+    public Validated<ErrorInput, List<StagedRow>> stageXlsxFile(MultipartFile file, Boolean withInvertSize) {
+        try {
+            InputStream inputStream = file.getInputStream();
+            OPCPackage opcPackage = OPCPackage.open(inputStream);
+            XSSFReader xssfReader = new XSSFReader(opcPackage);
+            ZipSecureFile.setMinInflateRatio(0.0d);
 
+            SurveyContentsHandler surveyContentsHandler = new SurveyContentsHandler((withInvertSize) ? longHeadersRef : shortHeadersRef);
```

Comment:
I know you didn't name it this but the option shouldn't be called withInvertSize - it wqould be better called withExtendedSizes - it applies to both fish and invert sizing and refers to extended sizes being used for both.  Its been a source of confusion for me up until recently until clarified by @bpasquer.  (Thanks Bene!)

Reply:
But you don't have to change this now.  I think its referred to as that in the api and must be referenced in the UI as well so would require a few changes.  Just ranting

Reply:
Will change the variable name in this PR but change the parameter name with the upload page PR.